creusot/src/backend/clone_map/elaborator.rs

```diff
@@ -409,21 +409,16 @@ impl<'a, 'tcx> Expander<'a, 'tcx> {
 
         let decls = match dep {
             Dependency::Type(_) => TyElab::expand(self, ctx, dep),
-            Dependency::Item(mut def_id, _) => {
+            Dependency::Item(def_id, _) => {
                 if ctx.is_logical(def_id) {
                     LogicElab::expand(self, ctx, dep)
                 } else if ctx.is_constant(dep.to_trans_id(ctx.tcx, self.param_env).unwrap()) {
                     LogicElab::expand(self, ctx, dep)
                 } else if matches!(ctx.def_kind(def_id), DefKind::Field) {
-                    if util::item_type(self.tcx, def_id) == ItemType::Field {
-                        def_id = self.tcx.parent(def_id);
-                    };
-                    let name = self.namer.insert(dep).qname();
-                    let modl = ctx.module_path(def_id);
-                    let name = if name.module.is_empty() { name } else { name.module_qname() };
+                    let Kind::Used(modl, alias, _) = self.namer.insert(dep) else { todo!() };
```

Comment:
Is this meant to be a `todo!` or unreachable?

Reply:
It was really a todo but the test suite doesn't reach it :shrug:

Reply:
I checked and it should be unreachable. Replaced with `panic!`